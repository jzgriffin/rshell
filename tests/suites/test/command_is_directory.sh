mkdir test_is_directory_dir
touch test_is_directory_directory
test -d test_is_directory_dir && echo a
test -d test_is_directory_directory && echo b
test -d test_not_is_directory_dir && echo c
test -d test_not_is_directory_directory && echo d
rm -rf test_is_directory_dir test_is_directory_directory
