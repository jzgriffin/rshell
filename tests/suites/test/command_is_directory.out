(True)
a
(False)
(False)
(False)
